uk.co.compucorp.civicrm.hremails/email-templates/src/assets/scss/components/_request.scss

```diff
@@ -15,12 +15,17 @@
 
 
 .request-comment {
-  padding: $callout-padding;
+
```

Comment:
it's just something that i'm used to do, it reads better to me. we never explicitly made it obligatory